### PR TITLE
[ADD] New module <mrp_bom_note>

### PR DESCRIPTION
**Pull request by oihane**

### CODE REVIEW

File: mrp_bom_note/README.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+MRP - BoM notes
+===============
+
+This module performs the following:
+
+1.- Creates the new field NOTES in MRP BoM list.
+
+2.- Creates the new field NOTES in MRP production, of type related with MRP BoM List.
```
Comment (pedrobaeza) [inline mrp_bom_note/README.rst]: Put OCA's template

Comment (rdeheele) [inline mrp_bom_note/README.rst]: I don't see "the new field NOTES in MRP production".

Comment (oihane) [inline mrp_bom_note/README.rst]: You're right, I've got to ask. Maybe is something that got lost between diferent versions, thanks for the point.

Comment (pedrobaeza) [inline mrp_bom_note/README.rst]: Finally it was split in 2 modules. The other is mrp_production_note

Comment (oihane) [inline mrp_bom_note/README.rst]: OK, so I'll update this README

**Comment by rdeheele** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by mathi123** [comment]
:+1:

**Pull request merged**